packages/ee/billing/frontend/runs-left-snackbar/runs-left-snackbar.component.ts

```diff
@@ -1,30 +1,31 @@
 import { ChangeDetectionStrategy, Component, Inject } from '@angular/core';
 import { MatSnackBarRef, MAT_SNACK_BAR_DATA } from '@angular/material/snack-bar';
 import { map, Observable } from 'rxjs';
-import { UsageService } from '@ee/usage/frontend/usage.service';
+import { BillingService } from '@ee/billing/frontend/billing.service';
 
 @Component({
   selector: 'app-runs-left-snackbar',
   templateUrl: './runs-left-snackbar.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RunsLeftSnackbarComponent {
-  runsStats$: Observable<{ runsCap: number, runsExecuted: number, daysLeftBeforeReset: number }>;
+  runsStats$: Observable<{ runsCap: number, runsExecuted: number, daysLeftBeforeReset: number, customerPortalUrl : string }>;
   constructor(
     public snackBarRef: MatSnackBarRef<RunsLeftSnackbarComponent>,
     @Inject(MAT_SNACK_BAR_DATA) public data: any,
-    private usageService: UsageService
+    private billingService: BillingService
   ) {
-    this.runsStats$ = this.usageService.getUsage().pipe(map(res => {
+    this.runsStats$ = this.billingService.getUsage().pipe(map(res => {
       return {
-        runsCap: res.metrics.steps.remaining + res.metrics.steps.consumed,
-        runsExecuted: res.metrics.steps.consumed,
-        daysLeftBeforeReset: Math.ceil(res.metrics.steps.nextResetInMs / 24 / 60 / 60 / 1000)
+        runsCap: res.plan.tasks,
+        runsExecuted: res.usage.consumedTasks,
+        daysLeftBeforeReset: Math.round((new Date(res.usage.nextResetDatetime ).getTime() - Date.now()) / 1000 / 60 / 60 / 24),
```

Comment:
Using plain javascript dates is a recipe for disasters, we already have `dayjs` as a dependency, why not use it?